core/src/main/java/de/flo56958/MineTinker/Data/ToolType.java

```diff
@@ -28,6 +29,44 @@
 	SWORD,
 	TRIDENT;
 
+	private static EnumMap<ToolType, List<Material>> tools = new EnumMap<>(ToolType.class);
+
+	static {
+		tools.put(ToolType.AXE, Arrays.asList(Material.WOODEN_AXE, Material.STONE_AXE, Material.IRON_AXE, Material.GOLDEN_AXE, Material.DIAMOND_AXE));
+		tools.put(ToolType.BOOTS, Arrays.asList(Material.LEATHER_BOOTS, Material.CHAINMAIL_BOOTS, Material.IRON_BOOTS, Material.GOLDEN_BOOTS, Material.DIAMOND_BOOTS));
+		tools.put(ToolType.BOW, Collections.singletonList(Material.BOW));
+		tools.put(ToolType.CHESTPLATE, Arrays.asList(Material.LEATHER_CHESTPLATE, Material.CHAINMAIL_CHESTPLATE, Material.IRON_CHESTPLATE, Material.GOLDEN_CHESTPLATE, Material.DIAMOND_CHESTPLATE));
+
+		if (NBTUtils.isOneFourteenCompatible()) {
+			tools.put(ToolType.CROSSBOW, Collections.singletonList(Material.CROSSBOW));
+		}
```

Comment:
There should be an else case with and empty List. ToolType.Crossbow gets used without checking for 1.14 but getting an empty list before and doing on that operations like .contains(). The current version will throw a NullPointerException! Giving a empty List would be better than checking every time the version.